lib/gtfsplus/util/index.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import {getGtfsPlusSpec} from '../../common/util/config'
+
+export function constructNewGtfsPlusRow (tableId: string): any {
```

Comment:
Please have a more specific return type than `any`.

Reply:
Not really sure what this return type should be.  It returns an object with fields that depend on which tableId is passed in.  Fields are set to `null`.

Reply:
If it can be one of numerous types you can use a the union type: https://flow.org/en/docs/types/unions/

Reply:
So, if I were to define multiple return types for this function, the whole point of defining the gtfsplus spec in this yml file would be lost: https://github.com/catalogueglobal/datatools-ui/blob/dev/gtfsplus.yml

We would just be duplicating the effort to describe this spec.

Reply:
ok, looking at this a bit more, I think the return type should be a key-value object where the values are `GtfsPlusField`s.

Example:

```javascript
{ [string]: GtfsPlusField }
```

Reply:
Actually, the return type should be `{ [string]: null }`.  Maybe you need another object for a GtfsPlusField or maybe you can modify the GtfsPlusField so that some of the values can be missing.